lib/fluent/plugin/out_webhdfs.rb

```diff
@@ -66,6 +66,10 @@ class Fluent::Plugin::WebHDFSOutput < Fluent::Plugin::Output
   config_param :kerberos, :bool, default: false
   desc 'kerberos keytab file'
   config_param :kerberos_keytab, :string, default: nil
+  desc 'Use delegation token while upload webhdfs or not'
+  config_param :reuse_delegation_token, :bool, default: false
```

Comment:
I think `renew_kerberos_delegation_token` is more suitable for the name to correspond with `renew_kerberos_delegation_token_time_hour`.